test/models/compute/test_managed_disk.rb

```diff
@@ -50,13 +50,19 @@ def test_save_method_response
 
   def test_destroy_method_true_response
     @service.stub :delete_managed_disk, true do
-      assert @managed_disk.destroy
+      assert @managed_disk.destroy(false)
     end
   end
 
   def test_destroy_method_false_response
     @service.stub :delete_managed_disk, false do
-      assert !@managed_disk.destroy
+      assert !@managed_disk.destroy(false)
+    end
+  end
+
+  def test_destroy_method_can_take_params_async
```

Comment:
Update this unit test to reflect the changes you make above. Again, you can reference server unit tests for better understanding.